frontend/src/components/buttons/OblongButton.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import styles from '../../styles/buttons/OblongButton.module.css';
+import { Link } from 'react-router-dom';
+
+interface ButtonProps {
+  text: string;
+  link: string;
+  color: string;
+  ariaLabel?: string;
+  externalLink?: boolean;
+}
+
+/*
+Color options: blue, green.
+If link is external, set externalLink to true.
+*/
+const OblongButton = (props: ButtonProps) => {
+  let buttonClass: string;
+
+  // Update class so stylesheet can change colors.
+  switch (props.color) {
+    case 'green':
+      buttonClass = `${styles.button} ${styles.green}`;
```

Comment:
Could put styles.button on line 33 instead for standard button animations/styling